packages/wallets/src/components/Base/WalletButton/WalletButton.scss

```diff
@@ -86,10 +86,10 @@ $size-map: (
             border-radius: 0.4rem;
         }
         &--md {
-            border-radius: 6.4rem;
+            border-radius: 0.8rem;
         }
         &--lg {
-            border-radius: 100%;
+            border-radius: 1.2rem;
```

Comment:
Since this is a base component wouldn't updating this border radius change all the components that are using it? Do all the components need the same updated value?

Reply:
@lubega-deriv, indeed, there are currently only two variations: 'sm' and 'md', with the majority of Buttons using 'sm'. I've introduced some new values while ensuring that the styles for other Buttons remain consistent.